### ISSUE TITLE
SSL/HTTPS support ?

### ISSUE THREAD

**Issue by GunGunGun** [alexkirsz/catalyst-proxy#2]
Hi Morhaus, first I want to thanks for creating such a great proxy.

These day every sites become SSL/HTTPS, so without HTTPS support seem Catalyst Proxy a little bit uneffective, example Youtube is now HTTPS, its googlevideo use HTTPS so I cannot accelerate video download speed using Catalyst, I hope you will add HTTPS support using something like Man in the middle method. Thanks!

**Comment by alexkirsz** [comment]
Yeah, I agree that catalyst is kind of useless without HTTPS support. I've been looking to add SSL support for a while but my first attempts at it were unfruitful.

**Comment by lsycxyj** [comment]
Any progress now?
